src/utils/formatDate.ts

```diff
@@ -0,0 +1,39 @@
+const dateTimeFormatLocale = 'en-GB'
+
+export function formatDateTime(date: Date) {
```

Comment:
* [ ] FYI Refactoring idea: Export a type for available formats and provide a single helper function.

```ts
export type DateFormat = 'dateTime' | 'fullDateTime' // | etc
export function formatDate(date: Date, format: DateFormat)
```